controllers/configuration_controller.go

```diff
@@ -409,6 +420,47 @@ func (r *ConfigurationReconciler) preCheck(ctx context.Context, configuration *v
 		meta.GitImage = "alpine/git:latest"
 	}
 
+	meta.ResourcesLimitsCPU = os.Getenv("RESOURCES_LIMITS_CPU")
```

Comment:
These environments should be set in the controller manifests like https://github.com/oam-dev/terraform-controller/blob/master/chart/templates/terraform_controller.yaml#L28-L37.